aws-opentelemetry-distro/src/amazon/opentelemetry/distro/patches/_bedrock_patches.py

```diff
@@ -217,6 +219,13 @@ def on_success(self, span: Span, result: _BotoResultT):
                 guardrail_id,
             )
 
+        guardrail_arn = result.get(_GUARDRAIL_ARN)
+        if guardrail_arn:
+            span.set_attribute(
+                AWS_BEDROCK_GUARDRAIL_ARN,
+                guardrail_arn,
+            )
```

Comment:
same to https://github.com/aws-observability/aws-otel-java-instrumentation/pull/905/files#r1796412547

Reply:
Sounds good, [will go with you and Zhonghao's recommendation](https://github.com/aws-observability/aws-otel-java-instrumentation/pull/905#discussion_r1797187133).